testsuite/pytests/sli2py_recording/test_multimeter_stepping.py

```diff
@@ -87,7 +87,8 @@ def build_net(model):
     """
 
     nest.ResetKernel()
-
+    if "eprop" in model:
+        nest.SetKernelStatus({"eprop_update_interval": 1.0, "eprop_update_interval_reset": True})
```

Comment:
> Could you motivate making these a kernel property rather than a property of the individual models?

They are needed by a synapse model and different neuron models, thus have to be globally available. Also, they are related to the task the network is performing, i.e., to the overarching experiment. Currently, there is no better place to put such parameters and we will thus leave them here for now in agreement with @heplesser.

Reply:
For my own understanding: should all the kernel parameters, e.g., also the `resolution` and `num_virtual_procs`, be set like this or only the e-prop related ones?
```
nest.eprop_update_interval =  1.0
nest.eprop_update_interval_reset = True
```

Reply:
Fixed in [commit 64f36151a](https://github.com/nest/nest-simulator/pull/2867/commits/64f36151a4b5f47fee929ea7937adc8f8719c756).

Reply:
We want to move away from the `GetStatus`/`SetStatus` methods with dictionaries requiring braces and quotes to the more object-oriented approach above. If you want to set several parameters at once (and a few need to be set together), the new way is
```
nest.set(resolution=0.5, local_num_threads=3)
```